.github/workflows/test_on_push.yml

```diff
@@ -230,10 +231,9 @@ jobs:
           cache: 'pip'
           cache-dependency-path: setup.py
 
-      - name: Install PyBaMM dependencies
+      - name: Install standard Python dependencies
         run: |
           pip install --upgrade pip wheel setuptools nox
-          pip install -e .[all,docs]
```

Comment:
Now that we have separate jobs for particular test groups (#3264) plus we don't run sessions sequentially either, we don't need this step anymore. The `nox` step will install PyBaMM with its dependencies anyway